tests/e2e/TestConstants.ts

```diff
@@ -84,46 +84,16 @@ export const TestConstants = {
     TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: process.env.TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME || 'petclinic-dev-environment',
 
     /**
-     * Catalog tile operator name on OperatorHub UI.
+     * Using 'single-host' strategy, "false" by default.
      */
-    TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME: process.env.TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME || 'eclipse-che-preview',
-
-    /**
-     * Operator logo name by installation using OperatorHub
-     */
-    TS_SELENIUM_OPERATOR_LOGO_NAME: process.env.TS_SELENIUM_OPERATOR_LOGO_NAME || 'Eclipse Che',
-
-    /**
-     * Name of namespace created on OCP for installation CHE by using OperatorHub UI.
-     */
-    TS_SELENIUM_INSTALL_PROJECT_NAME: process.env.TS_SELENIUM_INSTALL_PROJECT_NAME || '',
-
-    /**
-     * Update Channel name on 'Create Operator Subscription' page on OCP, "nightly" or "stable".
-     */
-    TS_OCP_OPERATOR_UPDATE_CHANNEL: process.env.TS_OCP_OPERATOR_UPDATE_CHANNEL || 'nightly',
-
-    /**
-     * Value of TLS Support property in the 'Create Che Cluster' yaml using OperatorHub.
-     */
-    TS_SELENIUM_VALUE_TLS_SUPPORT: process.env.TS_SELENIUM_VALUE_TLS_SUPPORT || 'true',
```

Comment:
Are you sure that we don't use this var?
Because I saw it a week ago in the CI job config

Reply:
Yes, I'm sure. This was create for _InstallCheByOperatorHub_ and now is unneeded and you could see in the 'intellij-smoke-che
```
e2eTestParameters: "-e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=true  -e TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=htpasswd  -e TS_SELENIUM_OCP_USERNAME=$openshiftDto.clusterAdminName -e TS_SELENIUM_OCP_PASSWORD=$openshiftDto.clusterAdminPassword -e TS_SELENIUM_MULTIUSER=true -e TS_SELENIUM_VALUE_TLS_SUPPORT=true -e TS_SELENIUM_VALUE_SELF_SIGN_CERT=true -e TS_SELENIUM_LOG_LEVEL=DEBUG -e TS_SELENIUM_W3C_CHROME_OPTION=true -e NODE_TLS_REJECT_UNAUTHORIZED=0 -e TS_SELENIUM_OCP_USERNAME=admin -e TS_SELENIUM_OCP_PASSWORD=crw4ever! -e TS_SELENIUM_CLICK_ON_VISIBLE_ITEM=60000",
```
But it isn't needed and should be deleted. I'm preparing MR in appropriate repo.